pkg/plugins/policies/core/matchers/testdata/matchedpolicies/meshgateways/03.policies.yaml

```
@@ -0,0 +1,90 @@
+---
+type: MeshAccessLog
+mesh: mesh-1
+name: mesh
+labels:
+  kuma.io/origin: zone
+  kuma.io/zone: zone-1
+spec:
+ targetRef:
+  kind: Mesh
+ to:
+  - targetRef:
+     kind: Mesh
+    default:
+      backends:
+        - type: File
+          file:
+            path: /mesh
+---
+type: MeshAccessLog
+mesh: mesh-1
+name: servicesubset
+labels:
+  kuma.io/origin: zone
+  kuma.io/zone: zone-1
+spec:
+ targetRef:
+  kind: MeshServiceSubset
+  name: edge-gateway
+  tags:
+    listener: two
+ to:
+  - targetRef:
+     kind: Mesh
+    default:
+      backends:
+        - type: File
+          file:
+            path: /servicesubset
+---
+type: MeshAccessLog
+mesh: mesh-1
+name: gateway
```

```suggestion
name: in-zone-2
```

Makes it easier to check the golden file because you are looking for zone-2 in something that runs in zone-1

Also the tests can have any name. So naming it something like "policies-in-other-zone" might be better than `03`

---

renamed